app/views/static_pages/invoice.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-12">
+      <div class="card card-default">
+        <div class="card-body">
+          <h3 id="card-text">Your charges</h3>
+          <hr>
+          <% @charges = Invoice.all.where(payment_id: current_person.user.payment.id)%>
+          <% @charges.each do |charge| %>
+            <div class="media mb-2">
+                <p><%= charge.description %></p>
+                <b><%= charge.amount %></b>
+                <p><%= charge.created_at %></p>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div></div></div>
```

Comment:
Please delete file ```bootstrap-datepicker-rails,``` from the repo as seen below. (I can't comment on an empty file)